### PR TITLE
New package: python-miniupnpc-2.0.2

### PR DESCRIPTION
**Pull request by faulesocke**
A Python UPnP library (bindings for miniupnpc).

### CODE REVIEW

File: srcpkgs/python-miniupnpc/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+version=2.0.2
+revision=1
+wrksrc="${pkgname#python-}-$version"
+build_style=python-module
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel miniupnpc-devel"
+depends="python"
+short_desc="Python2 bindings for miniupnpc, a UPnP library"
+maintainer="Faule Socke <github@socker.lepus.uberspace.de>"
+license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/python-miniupnpc/template]: use SPDX identifier

**Comment by faulesocke** [comment]
Fixed it (it actually never was GPL). The license looks similar to BSD but is slightly modified.

File: srcpkgs/python-miniupnpc/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+version=2.0.2
+revision=1
+wrksrc="${pkgname#python-}-$version"
+build_style=python-module
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel miniupnpc-devel"
+depends="python"
+short_desc="Python2 bindings for miniupnpc, a UPnP library"
+maintainer="Urs Schulz <voidpkgs@ursschulz.de>"
+license="custom"
```
Comment (maxice8) [inline srcpkgs/python-miniupnpc/template]: BSD-3-Clause

Comment (faulesocke) [reply srcpkgs/python-miniupnpc/template]: Done.

**Pull request closed without merging**